optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SingleStatistic.java

```diff
@@ -178,6 +178,18 @@ public void readCsvStatisticFile() {
         }
     }
 
+    public void retrievePointList() {
+        if (getCsvFile().exists() && pointList == null) {
+            initPointList();
+            readCsvStatisticFile();
+        }
+    }
+
+    public void persistPointList() {
```

Comment:
I didn't either, so I used "persist". As for "hibernate":
* -1 exactly because of the issue you raised with the Hibernate project, it even confused me for a second
* +1 as it does effectively describe removing the pointList from memory

I'll keep on the lookout for anything better and fall back to "hibernate" if I don't find anything.